test/Microsoft.ML.Core.Tests/UnitTests/TestCSharpApi.cs

```diff
@@ -254,7 +254,7 @@ public void TestCrossValidationBinaryMacro()
             }
         }
 
-        [Fact(Skip = "Differnce in Baseline files for netcoreapp 3.0")]
+        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.NetCore21))] // netcore3.0 output differs from Baseline
```

Comment:
Do these tests not need to run for full framework?

Reply:
the flag will be true for anything but .NetCore3.0, I will change the variable name to ```NotNetCore30```